### ISSUE TITLE
Initialization fails on parsing tags/hostname

### ISSUE THREAD

**Issue by gecube** [gravitational/teleport#42753]
Good day!

I got 

```
2024-06-11T11:25:04Z INFO  Starting Teleport with a config file version:15.4.0 config_file:/etc/teleport/teleport.yaml common/teleport.go:679
SDK 2024/06/11 11:25:04 WARN falling back to IMDSv1: operation error ec2imds: getToken, http response error StatusCode: 405, request to EC2 IMDS failed
2024-06-11T11:25:04Z INFO  Unsolicited response received on idle HTTP channel starting with "x\r\nDate: Tue, 11 Jun 2024 11:25:04 GMT\r\nContent-Type: application/text\r\nContent-Length: 136\r\nConnection: keep-alive\r\nEtag: a71b08e4dae55d068739a56531f9e1ce\r\n\r\nattributes/\ndescription\ndisks/\nhostname\nid\nmaintenance-event\nname\nnetwork-interfaces/\nservice-accounts/\ntags\nvendor/\nvirtual-clock/\nzone"; err=<nil>
ERROR: initialization failed
        strconv.ParseUint: parsing "fhmnhh6k6ud6rmdkefav": invalid syntax
```

in proxy pod logs.

I was expecting some configuration setting for disabling IMDS at all. Not just closing the #42073

I want to mention that we are enterprise customers so support level with resolution "we don't support Yandex cloud" makes me worrying about other clouds as well and the possible options of subscription prolongation.

**Comment by gecube** [comment]
Anyway - revert to 15.3.0 works like a charm.

**Comment by zmb3** [comment]
This is a duplicate of #42312 and the fix is in progress at #42696.

**Comment by gecube** [comment]
@zmb3 thank you very much, I will check the progress.
